app/controllers/comments_controller.rb

```diff
@@ -141,7 +141,26 @@ def update
     authorize @comment
     if @comment.update(permitted_attributes(@comment).merge(edited_at: Time.zone.now))
       Mention.create_all(@comment)
-      redirect_to URI.parse(@comment.path).path, notice: "Comment was successfully updated."
+
+      # The following sets variables used in the index view. We render the index
+      # view directly to avoid having to redirect which leads to caching issues.
```

Comment:
It works! I would link [this explanation](https://github.com/forem/forem/issues/10338#issuecomment-693401481) in the comment, mostly to clarify "caching issues"